graphql-schemas/schema.yml

```diff
@@ -1362,6 +1370,25 @@ confs:
   - { name: specs, type: SQSQueuesSpecs_v1, isRequired: true, isList: true }
   - { name: annotations, type: json }
 
+- name: NamespaceTerraformResourceSNSSubscription_v1
+  fields:
+  - { name: protocol, type: string, isRequired: true }
+  - { name: endpoint, type: string, isRequired: true}
+
+- name: NamespaceTerraformResourceSNSTopic_v1
+  interface: NamespaceTerraformResourceAWS_v1
+  fields:
+  - { name: provider, type: string, isRequired: true }
+  - { name: defaults, type: string, isResource: true, isRequired: true }
```

Comment:
You might want to make underlying json-schema consistent with GraphQL schema in terms of requirement of the fields. Otherwise bundle validation will succeed but query evaluation will lead into error as the field is NonNull with `isRequired` attribute in GraphQL world. 

I've documented this in SCHEMA.md if you need more clarification with it. 

Otherwise LGTM.

Reply:
So I actually wanted to make this field not required, but was getting error. There is no way around it since the interface requires it right?

Reply:
Correct. You can make this GraphQL schema consistent by adding `defaults` to `required` list in json-schema.

Reply:
Correct.
For some reason, github added a duplicate comment and when I removed it, it got rid of my comment here. 

Anyways, resolving this now.